client/pages/home.jsx

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Grid, Container, makeStyles } from '@material-ui/core';
+import Post from '../components/post';
+
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
+      display: 'flex',
+      justifyContent: 'center'
+    }
+  };
+});
 
 export default function Home() {
+  const classes = useStyles();
+
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    fetch('/api/home')
+      .then(res => res.json())
+      .then(posts => setPosts(posts));
+  }, []);
+
   return (
-    <div>
-    </div>
+    <Container className={classes.root}>
+      <Grid container spacing={1}>
+        {[...posts].reverse().map(post => (
```

Comment:
Instead of forcing your client to reverse the posts, just select them with an `order by` clause.